tests/test_coordinator.py

```diff
@@ -184,11 +224,14 @@ def test_wrong_participant():
 
 
 def test_duplicated_update_submit():
+    """[summary]
```

Comment:
We could make docstrings optionals in the `tests/` directory imo

Reply:
i strongly disagree, in my opinion the code that takes care of the quality of the rest of the code (meaning the tests) should adhere at least to the same standards as the other code. docstrings improve readability and therefor help to understand if the tests themselves are correctly written.

Reply:
That's fine then. However if we're to enforce this I think we shouldn't use dummy docstrings (see [my comments below](https://github.com/xainag/xain-fl/pull/210#discussion_r364622666))